lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -49,6 +68,31 @@ def _delete_environment(self, environment: Environment, log: Logger) -> None:
         with libvirt.open("qemu:///system") as qemu_conn:
             self._delete_nodes(environment, log, qemu_conn)
 
+    # Check what capabilities can be provided for the node.
+    def _create_node_capabilities(
+        self, log: Logger, node_space: schema.NodeSpace
+    ) -> schema.NodeSpace:
+        node_capabilities = schema.NodeSpace()
+        node_capabilities.name = "QEMU"
+        node_capabilities.node_count = 1
+        node_capabilities.core_count = 2
+        node_capabilities.memory_mb = 4096
+        node_capabilities.disk = schema.DiskOptionSettings()
+        node_capabilities.network_interface = schema.NetworkInterfaceOptionSettings()
+        node_capabilities.network_interface.max_nic_count = 1
```

Comment:
It needs to set `nic_count` too. the `max_nic_count` is used to limited how many nics can be created in some cases, which hot add nics.